.eslintrc.json

```diff
@@ -23,13 +22,17 @@
         "SharedArrayBuffer": "readonly"
     },
     "parserOptions": {
-        "ecmaVersion": 2020
+        "ecmaVersion": 2020,
+        "ecmaFeatures": {
+            "jsx": true
+        }
     },
     "plugins": [
         "requirejs"
     ],
     "rules": {
         "indent": ["error", 2, { "SwitchCase": 1 }],
+        "array-bracket-spacing": "off",
```

Comment:
Flipping this rule doesn't seem to have any effect in my VSC.

Either way, my preference would be for spacing because it's immediately obvious to my brain that's it's an array/destructure and not a lookup, e.g.
```jsx
classNamePrefixes={[ 'component__item', 'accordion__item' ]}
```

Reply:
From recollection, you have to restart vscode for it to be impactful. I've turned it off because we have a lot of variable array spacing, it wasn't my intention to open an argument on code formatting.